src/common/util/src/file_util.cpp

```diff
@@ -341,9 +341,9 @@ void ov::util::convert_path_win_style(std::string& path) {
 
 std::string ov::util::wstring_to_string(const std::wstring& wstr) {
 #    ifdef _WIN32
-    int size_needed = WideCharToMultiByte(CP_ACP, 0, &wstr[0], (int)wstr.size(), NULL, 0, NULL, NULL);
+    int size_needed = WideCharToMultiByte(CP_UTF8, 0, &wstr[0], (int)wstr.size(), NULL, 0, NULL, NULL);
```

Comment:
looks like it reverts this commit https://github.com/openvinotoolkit/openvino/commit/976298c41725ef7618e06a260b3f50f38e7d7a35

Reply:
@ilya-lavrenov  Another solution for catch dir, had synced with river. CP_ACP -> CP_UTF8 will be better.